native/cocos/application/CocosApplication.cpp

```diff
@@ -105,7 +105,7 @@ void CocosApplication::restart() {
 }
 
 void CocosApplication::close() {
-    _engine->close();
+    reinterpret_cast<UniversalPlatform*>(BasePlatform::getPlatform())->quit();
```

Comment:
The implementation may be more appropriate in SystemWindows, because createWindow and closeWindows should correspond to each other, create a window through createWindow, and close the application through closeWindow

I think the more appropriate exit process is this
ts(Game end)-> application.close->systemWindow.close()->receive the onclose message and call application.onClose->engine.close->platform to exit

Reply:
OK, I think I will move the implementation to SystemWindows